pynest/nest/lib/hl_api_info.py

```diff
@@ -89,9 +90,11 @@ def helpdesk():
     """
 
     if 'NEST_DOC_DIR' not in os.environ:
-        print(
-            'NEST help needs to know where NEST is installed.'
-            'Please source nest_vars.sh or define NEST_DOC_DIR manually.')
+        text = textwrap.dedent("""\
+            NEST help needs to know where NEST is installed.
+            Please source nest_vars.sh or define NEST_DOC_DIR manually
+        """)
```

Comment:
unnecessary use of dedent()

Reply:
@clinssen If I do not use `dedent()` the indenting of the printed text is a bit strange. Do you have any tips on how to get the best results for the printed text? The best I have managed is the one below. As you can see, there are empty spaces before the text starts, but at least the two sentences are on the same line. (This is without dedent())

I have the same problem with indenting for the print below as well.

![image](https://user-images.githubusercontent.com/21308850/84747451-8c8abd00-afb7-11ea-8d84-67aa43a112e6.png)

Reply:
Everything between the triple quotes will become part of the string, including indentation and newlines, so if we use triple-quoted strings, we should use 
```Python
        text = """\
NEST help needs to know where NEST is installed.
Please source nest_vars.sh or define NEST_DOC_DIR manually"""
```
instead of
```Python
        text = """\
            NEST help needs to know where NEST is installed.
            Please source nest_vars.sh or define NEST_DOC_DIR manually
        """
```
I find this a bit messy and would suggest to stick with single-quoted strings, as the strings in question aren't that long.